python-avd/pyavd/api/fabric_documentation/__init__.py

```diff
@@ -14,3 +17,4 @@ class FabricDocumentation:
     fabric_documentation: str = ""
     topology_csv: str = ""
     p2p_links_csv: str = ""
+    digital_twin: ClassVar[dict] = {}
```

Comment:
Please don't have classvars with mutables. It will mutate the class itself every time, which may break testing later.